.whitesource

```diff
@@ -0,0 +1,12 @@
+{
+  "scanSettings": {
+    "baseBranches": []
+  },
+  "checkRunSettings": {
+    "vulnerableCheckRunConclusionLevel": "failure",
+    "displayMode": "diff"
+  },
+  "issueSettings": {
+    "minSeverityLevel": "LOW"
+  }
+}
```

Comment:
is this suppose to be here?

Reply:
I've got whitesource enabled on the repo; it commits after the initial commit, but it's not relevant, no.